test/test-network/conf/25-nexthop-nothing.network

```diff
@@ -0,0 +1,74 @@
+[Match]
```

Comment:
Is this chunk supposed to be in this commit? This file is new and doesn't seem to be referenced anywhere.